src/structures/GroupChat.js

```diff
@@ -202,7 +202,7 @@ class GroupChat extends Chat {
             return window.Store.Invite.sendQueryGroupInviteCode(chatWid);
         }, this.id._serialized);
 
-        return code;
+        return code.code;
```

Comment:
Probably from another pr

Reply:
This was probably to keep my code running, it'll be reversed in the next commit.